babel.config.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable prefer-import/prefer-import-over-require */
+
+module.exports = (api) => {
+    api.cache(false);
+
+    return require('babel-preset-moxy/end-project')(api, {
+        modules: process.env.NODE_ENV === 'test' ? 'commonjs' : false,
+        react: true,
+        loose: false,
+        targets: {
+            // Maintain the node version as same as the one in Dockerfile
+            node: '12.6.0',
+            browsers: ['extends browserslist-config-google'],
```

Comment:
I think we should have this configuration point to the browsers' target as the **PostCSS** configuration. One option is to pass this to the `package.json` and have this file require the `package.json` and pass
```
browsers: require('./package.json').browserslist,
```

Reply:
One thing to consider is to change both eslint and stylelint presets to respect a brownish file or prop in package json.

Anyway, I agree to centralize this definition.